go.work

```diff
@@ -4,6 +4,7 @@ use (
 	.
 	./pkg/apimachinery
 	./pkg/apiserver
+	./pkg/promlib
```

Comment:
this also currently needs to be manually added to `GO_FILES` in the [Makefile](https://github.com/grafana/grafana/blob/8b9bc9a9193c4743a7bbc22f2313aef8546f5356/Makefile#L13)